frontend/__tests__/containers/MetricsStep/DeploymentFrequencySettings/DeploymentFrequencySettings.test.tsx

```diff
@@ -122,14 +158,21 @@ describe('DeploymentFrequencySettings', () => {
     });
     const listBox = within(getByRole('listbox'));
     await act(async () => {
-      await userEvent.click(listBox.getByText('mockOrgName'));
+      await userEvent.click(listBox.getByText('mockOrgName2'));
     });
 
     expect(updateDeploymentFrequencySettings).toHaveBeenCalledTimes(1);
   });
 
+  it('show render crews component when totalPipelineNumber is equal loadingCompletedNumber', () => {
```

Comment:
when -> given
is equal ??? 

Please write test description with biz aspect, not tech aspect

Reply:
ok